### ISSUE TITLE
[Legitimate Site Blocked] binances.club

### ISSUE THREAD

**Issue by blablo123** [MetaMask/eth-phishing-detect#6849]
Our site contains informational content.
Last year, our design was similar to the original Binance site. Maybe that's why you blocked us. Please unblock.

https://binances.club/

**Comment by devenblake** [comment]
Duplicate of #6696.
